docs/reference/random_sort.rst

```diff
@@ -0,0 +1,36 @@
+Random sort
+=======================
```

Comment:
Nitpick: Sphinx requires that lines defining headings be as long as the text itself. Otherwise, a warning during the build is emitted.